test/parallel/test-fs-realpath.js

```diff
@@ -201,11 +201,15 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); });
-  asynctest(fs.realpath, [entry], callback, function(err, result) {
-    assert.ok(err && true);
-    return true;
-  });
+  assert.throws(() => {
+    fs.realpathSync(entry);
+  }, common.expectsError({ code: 'ELOOP', type: Error }));
+  asynctest(
+    fs.realpath, [entry], callback, common.mustCall(function(err, result) {
+      assert.ok(err.path === entry);
```

Comment:
@Trott  Hi, I have made the modification as requested in another branch called test_11498. https://github.com/nodejs/node/pull/11622